ppa/archive/templates/archive/digitizedwork_detail.html

```diff
@@ -105,6 +105,45 @@ <h2 class="header">{{ object.title }}</h2>
         <dd>Lorem ipsum dolor sit amet, hinc esse senserit eos ad, postulant gubergren persequeris id duo. Dicat invenire eos ea. Vidit movet mea eu, soleat equidem voluptaria et duo.</dd>
     </dl> -->
 </div>
+
+{% if error %}
+  {# if something went wrong (e.g., bad query), display error message and nothing else #}
+  <p>{{ error }}</p>
+{% else %}
+  {% if page_highlights %}
+
+    {% for item in page_highlights %}
+        {% include 'archive/snippets/thumbnail_snippet.html' with id=item.srcid order=item.order item=item%}
```

Comment:
I think this isn't actually setting the highlight information properly for display in the snippet template.  Or, that is - it's not setting it in the context the same way it's being set for the archive view (which makes sense because in that case we're running two separate queries that we have to piece back together).